File: QE-Selenium-Java/Activity2.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {

    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();

        try {
            driver.get("https://training-support.net/webelements/login-form/");

            System.out.println(driver.getTitle());

            WebElement username = driver.findElement(By.cssSelector("#username"));
            username.sendKeys("admin");

            WebElement password = driver.findElement(By.cssSelector("#password"));
            password.sendKeys("password");

            WebElement submit = driver.findElement(By.cssSelector("button"));
            submit.click();

        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
}

File: QE-Selenium-Python/Activity2.py
# from selenium.webdriver.common.by import By
# from selenium import webdriver

# with webdriver.Firefox() as driver:
#     driver.get("https://training-support.net/webelements/login-form/")

#     print(driver.title)

#     element = driver.find_element(By.CSS_SELECTOR, "#username")
#     element.send_keys("admin")

#     password = driver.find_element(By.CSS_SELECTOR, "#password")
#     password.send_keys("password")

#     submit = driver.find_element(By.CSS_SELECTOR, "button[type='submit']")
#     submit.click()

#     driver.quit()

from selenium.webdriver.common.by import By
from selenium import webdriver
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/login-form/")
    print(driver.title)
    element =driver.find_element(By.XPATH,"//*[@id='username']")
    element.send_keys("devika")
    password=driver.find_element(By.ID,"password")
    password.send_keys("devika27")
    submit=driver.find_element(By.TAG_NAME,"button")
    submit.click()


